TexSoup/reader.py

```diff
@@ -163,11 +166,12 @@ def tokenize_string(text, delimiters=ALL_TOKENS):
     >>> print(tokenize_string(Buffer('0 & 1 \\\\\command')))
     0 & 1 \\
     """
-    result = ''
+#     result = ''
```

Comment:
There are a few comments - the new `join` operator seems better, so we can rid of these